pkg/config/control_plane.go

```diff
@@ -37,6 +37,8 @@ type ControlPlaneSpec struct {
 	Filestore ControlPlaneFileStore `json:"filestore"`
 	// The configuration of cache for control plane.
 	Cache ControlPlaneCache `json:"cache"`
+	// The configuration of insight collector
+	InsightCollectorConfig ControlPlaneInsightCollector
```

Comment:
Could you add tests for the control-plane config?
https://github.com/pipe-cd/pipe/blob/c7c50e6bbbf07df4068a3bf561bd21b7547d4fc9/pkg/config/testdata/control-plane/control-plane-config.yaml

Reply:
OK, thanks